### PR TITLE
[kmema-485] fixed pull view show problem when user changes network (wifi to 3g)

### PR DESCRIPTION
**Pull request by zurakb**
kmema-485

### CODE REVIEW

File: kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt [hunk 1]
```diff
@@ -73,8 +72,12 @@ class KmeDefaultPollEventHandler(
     }
 
     fun destroyValues(){
-        pollEnded.postValue(null)
+        destroyEndedValue()
         pollStarted.postValue(null)
         userAnsweredPoll.postValue(null)
     }
+
+    fun destroyEndedValue(){
```
Comment (AlexRubanskiyKaltura) [inline kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt:80]: Formatting issu. PLease add a space before `{`

File: kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt [hunk 2]
```diff
@@ -73,8 +72,12 @@ class KmeDefaultPollEventHandler(
     }
 
     fun destroyValues(){
```
Comment (AlexRubanskiyKaltura) [inline kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt:74]: Formatting issu. PLease add a space before `{`

File: kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt [hunk 3]
```diff
@@ -14,17 +16,14 @@ class KmeDefaultPollEventHandler(
     private val roomController: IKmeRoomController
 ) {
 
-    private val pollStarted = MutableLiveData<QuickPollStartedPayload>()
-    val pollStartedLiveData
-        get() = pollStarted as LiveData<QuickPollStartedPayload>
+    private val pollStarted = LiveEvent<QuickPollStartedPayload>()
+    val pollStartedLiveData get() = pollStarted.toSingleEvent()
```
Comment (AlexRubanskiyKaltura) [inline kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt:20]: As I remember .toSingleEvent() breaks single event logic. PLease check this on room login/logout and remove if so.

Comment (AlexRubanskiyKaltura) [reply kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt:20]: I mean should be like this (if breaks):

private val pollStarted = LiveEvent<QuickPollStartedPayload>()
val pollStartedLiveData get() = pollStarted

**Review by AlexRubanskiyKaltura — CHANGES_REQUESTED** [review]

File: kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt (hunk 3 above)
Comment (zurakb) [reply kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt]: Done

File: kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt (hunk 2 above)
Comment (zurakb) [reply kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt]: Done

File: kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt (hunk 1 above)
Comment (zurakb) [reply kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt]: Done

**Review by AlexRubanskiyKaltura — APPROVED** [review]

**Pull request merged**